lists_hw/MyLinkedList.java

```diff
@@ -0,0 +1,110 @@
+package ListLab;
+
+import java.util.Iterator;
+
+public class MyLinkedList<T> implements Iterable<T>{
+
+	private Node<T> head;
+	private int size = 0;
+
+	public int getSize() {
+		return size;
+	}
+
+	public boolean remove(int index) {
+		//to-do
+		if (index > size) {
+			return false;
+		}
+		Node<T>node = head;
+		Node<T>afterNode;
+		for (int i = 0; i < index-1; i++) {
+			node = node.getNext();
+		}
+		afterNode = node.getNext();
+		node.setNext(afterNode);
+		return true;
+
+	}
+
+	public T get(int index) {
+		//to-do
+		Node<T> nodeAtIndex = head;
+		for (int i = 0; i <index ; i++) {
+			nodeAtIndex = nodeAtIndex.getNext();
+		}
+		return nodeAtIndex.getData();
+	}
+
+	public void add(T obj) {
+		//to-do
+		if (head == null) {
+			head = new Node<>(obj);
+		}
+		Node<T> node = head;
+		while (node.getNext()!=null) {
+			node = node.getNext();
+		}
+		return node.getData();
```

Comment:
This method has a `void` return type, so you shouldn't be returning anything. Rather, you want to add "obj" as a new node at the end by doing `node.setNext(new Node(obj));` since `node` is currently refering to the last node.